bin/test_integration_rhel

```diff
@@ -1,23 +1,57 @@
 #!/bin/bash -ex
 
+function print_help() {
+  cat << EOF
+
+Run integration tests on RHEL container
+
+Usage: test_integration_rhel [options]
+
+    -h, --help            Shows this help message.
+
+    --rhel-version=7  Run in RHEL 7 container.
+
+    --rhel-version=8  Run in RHEL 8 container.
```

Comment:
fix indentation. also - i think we can have only an entry for `--rhel-version` and say that it is used to `Set the RHEL version to test (e.g --rhel-version=7)`

Reply:
@eranha  ^